File: OPENAPI-REST-API/openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/ServerIpmiLiveRequest.scala

package org.openapitools.client.model


case class ServerIpmiLiveRequest (
    /* Your IP Address you wish to connect to the IPMI system from. */
    _ip: String,
    /* Asset ID */
    _asset: Option[Integer]
)
object ServerIpmiLiveRequest {
    def toStringBody(var_ip: Object, var_asset: Object) =
        s"""
        | {
        | "ip":$var_ip,"asset":$var_asset
        | }
        """.stripMargin
}

File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/ServerIpmiLiveRequest.js
goog.provide('API.Client.ServerIpmiLiveRequest');

/**
 * Request body to setup an IPMI Live connection.
 * @record
 */
API.Client.ServerIpmiLiveRequest = function() {}

/**
 * Your IP Address you wish to connect to the IPMI system from.
 * @type {!string}
 * @export
 */
API.Client.ServerIpmiLiveRequest.prototype.ip;

/**
 * Asset ID
 * @type {!number}
 * @export
 */
API.Client.ServerIpmiLiveRequest.prototype.asset;

